src/java/picard/analysis/directed/TargetedPcrMetrics.java

```diff
@@ -17,79 +17,79 @@
+    /** Tracks the number of read pairs that pass the vendor's filter that do not have any observed duplicate reads (used to calc library size) */
     public long PF_SELECTED_UNIQUE_PAIRS;
 
-    /** PF Reads Aligned / PF Reads. */
+    /** Fraction of PF_READS that lack sample or optical duplicates and align to the reference genome (PF_UQ_READS_ALIGNED/PF_UNIQUE_READS) */
     public double PCT_PF_UQ_READS_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The total number of bases from PF_READS that align to the reference genome with mapping score > 0 */
     public long PF_BASES_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The total number of bases from unique PF_READS that align to the reference genome and have a mapping score > 0 */
     public long PF_UQ_BASES_ALIGNED;
 
-    /** The number of PF aligned amplified that mapped to an amplified region of the genome. */
+    /** The total number of bases from aligned reads that pass the vendor's filter (PF_BASES_ALIGNED) that map to an amplified region of the genome. */
     public long ON_AMPLICON_BASES;
 
-    /** The number of PF aligned bases that mapped to within a fixed interval of an amplified region, but not on a baited region. */
+    /** The total number of bases from aligned reads that have passed the vendor's filter (PF_BASES_ALIGNED), that map to within a fixed interval of an amplified region, but not on a baited region. */
     public long NEAR_AMPLICON_BASES;
 
-    /** The number of PF aligned bases that mapped to neither on or near an amplicon. */
+    /** The total number of PF aligned bases (PF_BASES_ALIGNED) that do not map to an amplicon or regions nearby an amplicon. */
     public long OFF_AMPLICON_BASES;
 
-    /** The number of PF aligned bases that mapped to a targeted region of the genome. */
+    /** The total number of bases from aligned reads that pass the vendor's filter (PF_BASES_ALIGNED) and map to a targeted region of the genome. */
     public long ON_TARGET_BASES;
 
-    /** The number of PF aligned bases that are mapped in pair to a targeted region of the genome. */
+    /** The total number of bases from read pairs that pass the vendor's filter and map to a targeted region of the genome. */
     public long ON_TARGET_FROM_PAIR_BASES;
 
-    /** On+Near Amplicon Bases / PF Bases Aligned. */
+    /** The fraction of bases from PF-aligned reads that map to or near an amplicon (NEAR_AMPLICON_BASES + ON_AMPLICON_BASES)/(PF_BASES_ALIGNED). */
     public double PCT_AMPLIFIED_BASES;
 
-    /** The percentage of aligned PF bases that mapped neither on or near an amplicon. */
+    /** The fraction of bases from PF-aligned reads that mapped neither on to or near an amplicon [PF_BASES_ALIGNED- (NEAR_AMPLICON_BASES + ON_AMPLICON_BASES)]/(PF_BASES_ALIGNED) */
```

Comment:
use one kind of parenthesis, if you need to convert a + to a -, you can do that.